packages/prisma/schema.prisma

```diff
@@ -564,8 +568,10 @@ model BookingReference {
   externalCalendarId         String?
   deleted                    Boolean?
 
-  credential   Credential? @relation(fields: [credentialId], references: [id], onDelete: SetNull)
-  credentialId Int?
+  credential    Credential?           @relation(fields: [credentialId], references: [id], onDelete: SetNull)
+  credentialId  Int?
+  delegatedTo   DomainWideDelegation? @relation(fields: [delegatedToId], references: [id], onDelete: Cascade)
```

Comment:
I don't see why I called it `delegatedToId` here and `domainWideDelegationCredentialId` in DestinationCalendar and SelectedCalendar. Seems like a miss from my side.

I think it should be quick to change it to `domainWideDelegationCredentialId` and `domainWideDelegationCredential`. Also, make sure to regenerate migrations to have no mention of delegatedTo in there

Reply:
> I don't see why I called it delegatedToId here and domainWideDelegationCredentialId in DestinationCalendar and SelectedCalendar. Seems like a miss from my side.

Actually i added this field. let me fix the name